modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/catalog/resource/v1_0/test/ProductResourceTest.java

```diff
@@ -165,6 +212,10 @@ private Product _addCPDefinition(Product product) throws Exception {
 				productId = cpDefinition2.getCProductId();
 				productType = cpDefinition2.getProductTypeName();
 				shortDescription = product.getShortDescription();
+				slug = cpDefinition2.getUrlTitleMap(
+				).get(
+					LocaleUtil.fromLanguageId("en_US")
+				);
```

Comment:
I would simplify it this way

`slug = cpDefinition2.getURL(LocaleUtil.toLanguageId(siteDefaultLocale));`

as for example is done here
[ProductDTOConverter.java#L108](https://github.com/eduardo-diniz/liferay-portal/blob/616d8faa0a4a9bc0508b734044661999da880963/modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java#L108)

Reply:
thanks :rocket: